docs/platforms/powershell/data-management/data-collected.mdx

```diff
@@ -0,0 +1,79 @@
+
+To start sending logged-in user information, set `$_.SendDefaultPii = true` in the `Start-Sentry` command.
+
+## Users' IP Address
+
+By default, the Sentry SDK doesn't send the user's IP address. Even if enabled, whether you're able to send the user's IP address or not, will depend on the integrations you enable in Sentry's SDK. Most integrations won't set the user's IP address at all.
+
+To enable sending the user's IP address, set `$_.SendDefaultPii = true` in the `Start-Sentry` command.
+
+## Request URL
+
+The full request URL of outgoing and incoming HTTP requests is **always sent to Sentry**. Depending on your application, this could contain PII data.
+
+## Request Query String
+
+The full request query string of outgoing and incoming HTTP requests is **always sent to Sentry**. Depending on your application, this could contain PII data.
+
+## Request Body
+
+The request body of incoming HTTP requests can be sent to Sentry. Whether it's sent or not, depends on the type and size of request body as described below:
+
+- **The type of the request body:**
+  - JSON and form bodies are sent
+  - Raw request bodies are always removed
+  - Uploaded files in the request bodies are never sent to Sentry
+- **The size of the request body:** There's a ["max_request_body_size"](/platforms/python/configuration/options/#max-request-body-size) option that's set to `medium` by default. This means that larger request bodies aren't sent to Sentry.
+
+If you want to prevent bodies from being sent to Sentry altogether, set `max_request_body_size` to `"never"`.
+
+## Source Context
+
+When an unhandled exception is sent to Sentry, a snapshot of the source code surrounding the line where the error originates is sent with it.
+
+To opt out of sending this source context to Sentry, set `include_source_context` to `False`.
+
+## Local Variables In Stack Trace
+
+When unhandled errors and exceptions are sent to Sentry, the names and values of local variables that were set when the errors occurred are sent at the same time.
+
+You can stop sending local variables to Sentry by setting `include_local_variables=False` in the `sentry_sdk.init()` call.
+
+## SQL Queries
+
+While SQL queries are sent to Sentry, neither the full SQL query (`UPDATE app_user SET password='supersecret' WHERE id=1;`), nor the values of its parameters will ever be sent. A parameterized version of the query (`UPDATE app_user SET password='%s' WHERE id=%s;`) is sent instead.
+
+## LLM Inputs And Responses
+
+When using Sentry in your AI apps, the SDK by default won't add data like LLM inputs and responses to spans. To start recording these, add `send_default_pii=True` to your `sentry_sdk.init()` call.
+
+Most AI integrations have an additional parameter to control whether prompts should be included called `include_prompts`. See the <PlatformLink to="/integrations/#ai">documentation for the specific AI framework</PlatformLink> for more information.
```

Comment:
irrelevant in .net

Is this copied from python?

Reply:
It may eventually be relevant for .NET (if we implemented something like https://github.com/getsentry/sentry-dotnet/issues/3762)... but not yet, no.